.github/workflows/publish-application-packages.yml

```diff
@@ -0,0 +1,45 @@
+name: Zarf Application Package Publishing
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+    inputs:
+      branchName:
+        description: "Branch to build the packages from"
+        required: true
+        default: "main"
+
+jobs:
+  publish-packages:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: "Checkout Repo"
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ github.event.inputs.branchName }}
+
+      - name: Setup golang
+        uses: ./.github/actions/golang
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/node
+
+      - name: Install The Latest Release Version of Zarf
+        uses: defenseunicorns/setup-zarf@main
+
+      - name: "Login to GHCR"
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
+        with:
+          registry: ghcr.io
+          username: dummy
+          password: ${{ github.token }}
+
+      - name: Build And Publish Application Packages
+        # TODO: Add package signing to this step
```

Comment:
TODO

Reply:
This TODO is likely fine until we unblock things.

Reply:
Should be resolved 🤞